src/components/SignUp/SignUp.js

```diff
@@ -115,10 +135,10 @@ function SignUp() {
                     >
                         {isLoading ? <Spinner /> : "sign up"}
                     </button>
+                    {error.api && <Error />}
```

Comment:
error.emailInUse is broken:
- error.emailInUse never gets set (there is no `setError(error => {emailInUse: 'errormessage'})` anywhere
- line 147 mixes error.emailInUse & error.api
- both don't show the icon (<Error/> component isn't used)